controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -375,7 +374,7 @@ def setUp(self):
 
         self.mediator.maximal_volume_size_in_bytes = 10
         self.mediator.minimal_volume_size_in_bytes = 2
-
+        self.mediator._identifier = "id123"

```

Comment:
could we use a more descriptive value? e.g. `"system_id_stub"`

Reply:
done